lib/gitsh/interactive_runner.rb

```diff
@@ -23,6 +26,7 @@ def run
       setup_readline
       handle_window_resize
       greet_user
+      load_gitshrc
```

Comment:
Don't we want to load the rc file before greeting the user? Might be good for i.e turning of the greeting message?